newIDE/app/src/SceneEditor/index.js

```diff
@@ -366,6 +367,22 @@ export default class SceneEditor extends Component {
     );
   };
 
+  _onObjectSelected = selectedObjectName => {
```

Comment:
selectedObjectName can be an empty string in case you unselect a selected object.
This means that we have selectedObjectNames that will be an array of a single empty string, which is wrong. I'll fix this.